pkg/models/devops/devops.go

```diff
@@ -158,90 +160,145 @@ func (d devopsOperator) UpdateDevOpsProject(workspace string, project *v1alpha3.
 	return d.ksclient.DevopsV1alpha3().DevOpsProjects().Update(project)
 }
 
-func (d devopsOperator) ListDevOpsProject(workspace string) (*v1alpha3.DevOpsProjectList, error) {
-	return d.ksclient.DevopsV1alpha3().DevOpsProjects().List(metav1.ListOptions{})
+func (d devopsOperator) ListDevOpsProject(workspace string, limit, offset int) (models.PageableResponse, error) {
```

Comment:
we are switching to api.ListResult `pkg/api/types.go`, please use api.ListResult instead of models.PageableResponse

Reply:
fixed